src/background/engineManager.ts

```diff
@@ -544,6 +551,46 @@ export class EngineManager {
     }
     return "ok";
   }
+
+  /**
+   * サードパーティ向けの設定ファイルを書き出す
+   */
+  private lock = new AsyncLock({
+    timeout: 1000,
+  });
+  private lockKey = "write";
+
+  async writeEngineInfoFor3rdParty() {
```

Comment:
ドキュメントの場所は関数の真上のが良いかもです（関数の上に書くとVSCodeでマウスホバーしたときに出てくる）
```suggestion
  private lock = new AsyncLock({
    timeout: 1000,
  });
  private lockKey = "write";

  /**
   * サードパーティ向けの設定ファイルを書き出す
   */
  async writeEngineInfoFor3rdParty() {
```